rust/agama-dbus-server/src/questions.rs

```diff
@@ -103,7 +110,17 @@ trait AnswerStrategy {
 /// AnswerStrategy that provides as answer the default option.
 struct DefaultAnswers;
 
+impl DefaultAnswers {
+    pub fn id() -> u8 {
+        1
```

Comment:
NP: I think this can be a field of the struct.

Reply:
reason why it is also class method is to allow detection of strategy like below that instance is DefaultAnswers